addons/openebs/3.3.0/install.sh

```diff
@@ -40,17 +40,45 @@ function openebs() {
     openebs_maybe_migrate_from_rook
 }
 
-# if rook-ceph is installed but is not specified in the kURL spec, migrate data from
+# if rook-ceph is installed but is not specified in the kURL spec, migrate data from 
 # rook-ceph to OpenEBS local pv hostpath
 function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
+            prompt_migrate_from_rook
             rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS"
             DID_MIGRATE_ROOK_PVCS=1 # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
 
+function prompt_migrate_from_rook() {
+    local ceph_disk_usage_total
+    local rook_ceph_exec_deploy
+
+    if kubectl get deployment -n rook-ceph rook-ceph-tools &>/dev/null; then
+        rook_ceph_exec_deploy=rook-ceph-tools
+    fi
+    ceph_disk_usage_total=$(kubectl exec -n rook-ceph deployment/$rook_ceph_exec_deploy -- ceph df | grep TOTAL | awk '{ print $8$9 }')
```

Comment:
i think shellcheck will complain here

```suggestion
    ceph_disk_usage_total=$(kubectl exec -n rook-ceph "deployment/$rook_ceph_exec_deploy" -- ceph df | grep TOTAL | awk '{ print $8$9 }')
```

Reply:
actually, `shellcheck` did not flag this as an issue